src/System.Private.CoreLib/shared/System/Buffers/Text/Utf8Parser/ParserHelpers.cs

```diff
@@ -70,5 +70,29 @@ public static bool TryParseThrowFormatException<T>(out T value, out int bytesCon
             value = default;
             return TryParseThrowFormatException(out bytesConsumed);
         }
+
+        public static bool TryGetNextTwoDigits(ReadOnlySpan<byte> source, int prevSourceIndex, out int sourceIndex, out int value)
+        {
+            if (source.Length - prevSourceIndex < 2)
+            {
+                sourceIndex = prevSourceIndex;
+                value = default;
+                return false;
+            }
+
+            uint digit1 = source[prevSourceIndex++] - 48u;
+            uint digit2 = source[prevSourceIndex++] - 48u;
```

Comment:
Nit: please use (uint)'0' instead; it's otherwise not clear what is the relevance of "48"